package/CHANGELOG

```diff
@@ -28,6 +26,9 @@ Fixes
   * Testsuite does not any more matplotlib.use('agg') (#2191)
   * In ChainReader, read_frame does not trigger change of iterating position. 
     (Issue #2723, PR #2815)
+  * TRZReader now checks `n_atoms` during initilization. (PR #2820)
```

Comment:
I might be missing something obvious, but the check is happening in `_read_next_timestep` right? So technically it's not limited to initialization but every step. It might be more accurate here to just say "checks `n_atoms` on reading"?

Reply:
Could you also add the issue number here?